inspirehep/modules/search/api.py

```diff
@@ -111,6 +110,24 @@ def default_fields(self):
         """What fields to use when no keyword is specified."""
         return ['_all']
 
+    @staticmethod
+    def citations(record, page=1, size=10):
```

Comment:
what about passing only the ``recid`` it's a bit cleaner, (since we are touching the code :P)

Reply:
WDYT?

Reply:
I asked you about that when I was taking this task and you decided that we shouldn't change this :P